javascript/central-login-pingone/README.md

```diff
@@ -0,0 +1,58 @@
+# Central login
+
+The ForgeRock SDK provides an option for using the Authorization Code Flow (with PKCE) with a centralized login application.
+
+For a non-authenticated user, use the `login: redirect` parameter of the `TokenManager` class to request OAuth/OIDC tokens. 
+
+This instructs the SDK to redirect the user to the login application that uses either the ForgeRock platform, or PingOne. 
+
+After successful authentication, the SDK redirects the user back to this sample application to obtain OAuth/OIDC tokens and complete the centralized login flow.
+
+To configure this sample, follow the steps in [Getting Started](#getting-started).
+
+Then, run the sample app as follows:
+```
+npm run start:central-login
+``` 
+
+### Instructions
+
+* [ForgeRock server](#forgerock-server)
```

Comment:
If this one applies to both PingOne and PingAM/AIC, did we decide to use "OIDC" as the "server" name?
I can't quite remember what convention we leaned towards.

Reply:
I have 2 separate ones for "Central-Login" to demonstrate both options. As part of another ticket I will add a generic one for React

Reply:
I deleted the Forgeorck / Ping references from the specific readme files